packages/rocketchat-markdown/markdown.js

```diff
@@ -77,4 +77,8 @@ RocketChat.callbacks.add('renderMessage', MarkdownMessage, RocketChat.callbacks.
 if (Meteor.isClient) {
 	Blaze.registerHelper('RocketChatMarkdown', text => Markdown.parse(text));
 	Blaze.registerHelper('RocketChatMarkdownUnescape', text => Markdown.parseNotEscaped(text));
+	Blaze.registerHelper('RocketChatMarkdownWithoutP', (text) => {
```

Comment:
I think `RocketChatMarkdownInline` communicate better the result we're getting from this helper.